config/config.md

```diff
@@ -5,6 +5,20 @@
 checker_severity_map.json file contains a mapping between a
 checker name and a severity level. Severity levels can be found in the shared.thrift file.
 
+The following severity levels are defined:
+
+STYLE: A true positive indicates that the source code is against a specific coding guideline or could improve readability.
+Example: LLVM Coding Guideline: Do not use else or else if after something that interrupts control flow (break, return, throw, continue).
+
+LOW: A true positive indicates that the source code is hard to read/understand or could be easily optimized.
+Example: Unused variables, Dead code.
+
+MEDIUM: A true positive indicates that the source code that may not cause a run-time error (yet), but against intuition and hence prone to error.
+Example: Redundant expression in a condition. 
+
+HIGH: A true positive indicates that the source code will cause a run-time error.
+  Example of this category: out of bounds array access, division by zero, memory leak.
+
```

Comment:
We also have `CRITICAL` severity level which is not used in `checker_severity_map.json` file but listed and can be used in the gui:
https://github.com/Ericsson/codechecker/blob/520f27f83edbce6ae151a1d95601ff1f99f1c259/api/v6/report_server.thrift#L63
Do not we want to mention this severity level here?